backend/app/core/auth.py

```
@@ -1,177 +1,133 @@
-        return user
+        stmt = select(User).where(
+            User.id == user_uuid,
+            User.is_active.is_(True),
+            User.is_deleted.is_(False)
+        )
+
+        try:
+            result = await self.db.execute(stmt)
+            user = result.scalar_one_or_none()
+            return user
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)